### PR TITLE
Update SQL.txt

### PR DESCRIPTION
**Pull request by krisderycke**

### CODE REVIEW

File: SQL.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ CREATE DATABASE IF NOT EXISTS `My Notes`;
 CREATE TABLE `ID`(
```
Comment (nicplackle) [inline SQL.txt:3]: Name 'ID' should not be there.

**Comment by MerelVanPuymbroeck** [comment]
Why two times  Note_Context VARCHAR (2000),  NOTE_Context VARCHAR (800) and 2 different lenghts?

**Comment by krisderycke** [comment]
> Why two times Note_Context VARCHAR (2000), NOTE_Context VARCHAR (800) and 2 different lenghts?

the varchar(2000) was what i had deleted before

**Comment by iliastic** [comment]
Does the date shows automatically or do you have to write it manually?

**Pull request merged**